pyinfra/facts/server.py

```diff
@@ -390,7 +390,7 @@ def process(output):
                     'comments': current_comments,
                 }
             else:
-                minute, hour, day_of_month, month, day_of_week, command = line.split(' ', 5)
+                minute, hour, day_of_month, month, day_of_week, command = line.split(maxsplit=5)
```

Comment:
```suggestion
                minute, hour, day_of_month, month, day_of_week, command = line.split(None, 5)
```

This should fix the error - the `maxsplit` argument is positional in 2.7 not a keyword. Looking forward to being rid of python 2.7 (https://github.com/Fizzadar/pyinfra/issues/329)!